understanding/20/low-or-no-background-audio.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Low or No Background Audio</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Prerecorded speech is not disrupted by background sound</dd>
-         <dt>Author task</dt><dd>Avoid or lessen background sound, or let users turn it off</dd>
+         <dt>Goal</dt><dd>Prerecorded speech is not disrupted by background sound.</dd>
+         <dt>What to do</dt><dd>Avoid or lessen background sound, or let users turn it off.</dd>
+         <dt>Why it's important</dt><dd>People who are hard of hearing may have difficulty distinguishing speech from music and other sounds.</dd> 
```

Comment:
@rainbreaw commented
> Consider including people with audio processing disabilities.

Reply:
This is easy to incorporate, but there is no mention of the Benefit or need in the Understanding document. Please create a PR for that, and free to provide an update to the Why at the same time. Suggestion:
> People who are hard of hearing or have audio processing disabilities may have difficulty distinguishing speech from music and other sounds.

Reply:
@GreggVan said
> +1 but also add.   Having clear speech also allows automatic speech recognition for auto-captioning, translation, simplification and many other accessibility strategies.

Reply:
As with all these benefits not captured in the Understanding document, I have added them to a new issue to track https://github.com/w3c/wcag/issues/3429